tests/queries/0_stateless/02227_test_create_empty_sqlite_db.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+# See 01658_read_file_to_string_column.sh
+user_files_path=$(clickhouse-client --query "select _path,_file from file('nonexist.txt', 'CSV', 'val1 char')" 2>&1 | grep Exception | awk '{gsub("/nonexist.txt","",$9); print $9}')
+
+function cleanup()
+{
+    ${CLICKHOUSE_CLIENT} --query="DROP DATABASE IF EXISTS ${CURR_DATABASE}"
+    rm -r "${DB_PATH}"
+}
+trap cleanup EXIT
+
+export CURR_DATABASE="test_01889_sqllite_${CLICKHOUSE_DATABASE}"
+
+DB_PATH=${user_files_path}/${CURR_DATABASE}_db1
+
+${CLICKHOUSE_CLIENT} --multiquery --multiline --query="""
```

Comment:
Single quote `"` would work as well
(Actually `"""..."""` is not a special syntax but concatenation `"" + "..." + ""`)